yazi-plugin/preset/utils.lua

```diff
@@ -30,6 +30,7 @@ end
 
 function utils.readable_path(path)
 	local home = os.getenv("HOME")
+  home = home and home or os.getenv("USERPROFILE")
```

Comment:
Maybe we can just `local home = os.getenv("USERPROFILE") or os.getenv("HOME")`?

Reply:
yup, it's cleaner